tests/fuzz/README.md

```diff
@@ -29,7 +29,7 @@ is recommended to use clang compiler.
 ```bash
 (\
 mkdir -p build && cd build && \
-CC=clang CXX=clang++ cmake .. -DENABLE_FUZZING=ON -DENABLE_SANITIZER=ON -DTREAT_WARNING_AS_ERROR=OFF && \
+CC=clang-10 CXX=clang++-10 cmake .. -DENABLE_FUZZING=ON -DENABLE_SANITIZER=ON -DTREAT_WARNING_AS_ERROR=OFF && \
```

Comment:
Please exclude tests/fuzz/README.md file from this PR. The current from the master is correct.

Reply:
fixed